### PR TITLE
docs: upload export api specification

### PR DESCRIPTION
**Pull request by sthirthala**
docs: upload export api specification

### CODE REVIEW

**Review by wenytang-ms — DISMISSED** [review]

File: media/export-api.gif.gif
Comment (formulahendry) [inline media/export-api.gif.gif:1]: export-api.gif.gif -> export-api.gif

**Review by formulahendry — CHANGES_REQUESTED** [review]
Change the naming~

**Review by wenytang-ms — COMMENTED** [review]
seems the gif image invalid

File: media/export-api.gif
Comment (formulahendry) [inline media/export-api.gif:1]: It seems the gif is invalid:
image

**Review by wenytang-ms — APPROVED** [review]

**Review by formulahendry — APPROVED** [review]

**Pull request merged**